lib/game.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/player'
+
+class Game
+  WINNING_SET = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+    [1, 4, 7],
+    [2, 5, 8],
+    [3, 6, 9],
+    [1, 5, 9],
+    [3, 5, 7]
+  ].freeze
+
+  attr_accessor :player1, :player2, :game_arr
```

Comment:
- [ ] You should use  attr_readers or attr_writers. Only using attr_accessors it allows writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))